cost/azure/rightsize_netapp_files/README.md

```diff
@@ -0,0 +1,26 @@
+# Azure Rightsize NetApp Files
+
+## What it does
```

Comment:
Seems like you are using managed disks description. I think this should be related to pools and volumes